test/ae_mdw_web/controllers/activities_controller_test.exs

```diff
@@ -1023,180 +1023,265 @@ defmodule AeMdwWeb.ActivitiesControllerTest do
+      next_account_id = :aeser_id.create(:account, next_account_pk)
+      contract_id = :aeser_id.create(:contract, contract_pk)
+      height = 398
+      mbi = 2
+      kb_hash = TS.key_block_hash(0)
+      mb_hash = TS.micro_block_hash(0)
+      enc_mb_hash = Enc.encode(:micro_block_hash, mb_hash)
+
+      {:ok, aetx} =
+        :aec_spend_tx.new(%{
+          sender_id: account_id,
+          recipient_id: next_account_id,
+          amount: 2,
+          fee: 3,
+          nonce: 4,
+          payload: ""
+        })
+
+      {:spend_tx, tx} = :aetx.specialize_type(aetx)
+
+      {:ok, contract_call4_aetx} =
+        :aect_call_tx.new(%{
+          caller_id: account_id,
+          nonce: 2,
+          contract_id: contract_id,
+          abi_version: 2,
+          fee: 1,
+          amount: 1,
+          gas: 1,
+          gas_price: 1,
+          call_data: ""
+        })
+
+      {:contract_call_tx, contract_call4_tx} = :aetx.specialize_type(contract_call4_aetx)
+
+      {:ok, contract_call5_aetx} =
+        :aect_call_tx.new(%{
+          caller_id: next_account_id,
+          nonce: 2,
+          contract_id: contract_id,
+          abi_version: 2,
+          fee: 1,
+          amount: 1,
+          gas: 1,
+          gas_price: 1,
+          call_data: ""
+        })
+
+      {:contract_call_tx, contract_call5_tx} = :aetx.specialize_type(contract_call5_aetx)
+
+      store =
+        empty_store()
+        |> Store.put(Model.Field, Model.field(index: {:contract_call_tx, 1, account_pk, 1}))
+        |> Store.put(Model.Tx, Model.tx(index: 1, block_index: {height, mbi}, id: "tx-hash1"))
+        |> Store.put(Model.Field, Model.field(index: {:contract_call_tx, 1, account_pk, 2}))
+        |> Store.put(Model.Tx, Model.tx(index: 2, block_index: {height, mbi}, id: "tx-hash2"))
+        |> Store.put(
+          Model.Field,
+          Model.field(index: {:contract_create_tx, nil, next_account_pk, 4})
+        )
+        |> Store.put(Model.Block, Model.block(index: {height, -1}, hash: kb_hash))
+        |> Store.put(Model.Block, Model.block(index: {height, mbi}, hash: mb_hash))
+        |> Store.put(
+          Model.AexnTransfer,
+          Model.aexn_transfer(index: {:aex9, account_pk, 4, next_account_pk, 1, 1})
+        )
+        |> Store.put(Model.Tx, Model.tx(index: 4, block_index: {height, mbi}, id: "tx-hash4"))
+        |> Store.put(
+          # Skipped
+          Model.RevAexnTransfer,
+          Model.aexn_transfer(index: {:aex141, account_pk, 5, next_account_pk, 2, 2})
+        )
+        |> Store.put(
+          Model.AexnTransfer,
+          Model.aexn_transfer(index: {:aex141, next_account_pk, 5, account_pk, 2, 2})
+        )
+        |> Store.put(Model.Tx, Model.tx(index: 5, block_index: {height, mbi}, id: "tx-hash5"))
+        |> Store.put(
+          # Skipped
+          Model.TargetKindIntTransferTx,
+          Model.target_kind_int_transfer_tx(index: {account_pk, "reward_oracle", {height, 6}, 6})
```

Comment:
Made sense for me to skip due to the passive role while receiving the reward, even when the oracle executes the query itself. Likewise might be the case to add the `oracle_query_tx` with the `oracle_id` position [here](https://github.com/aeternity/ae_mdw/blob/6631dee6a7449240e8c3b59f3aabedd68719e26a/lib/ae_mdw/fields.ex#L21) as the oracle receives the request to execute the query.

Reply:
Added it as an excluded pair, thanks